go.mod

```diff
@@ -36,7 +35,7 @@ require (
 
 replace (
 	// Pin RHsyseng library versions
-	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20200414161125-135bc9b52976
+	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20200414162234-f1e798e60303
```

Comment:
@swati-kale this change is unnecessary as we're already using the latest "non-merge" commit. please revert the module changes and we can get this merged.